src/store/modules/order/mutation-types.ts

```diff
@@ -0,0 +1,3 @@
+export const SN_DRAFT_ORDER = 'order'
```

Comment:
```suggestion
export const SN_ORDER = 'order'
```
@adityasharma7 Sir, what do you think?

Reply:
Our module is named as order, so I think we should call out mutation root as order.